tests/pytorch/test_nn.py

```diff
@@ -1666,3 +1666,22 @@ def test_MetaPath2Vec(idtype):
     model = model.to(dev)
     embeds = model.node_embed.weight
     assert embeds.shape[0] == g.num_nodes()
+
+def test_LaplacianPosEnc():
```

Comment:
you may use `@pytest.mark.parametrize` to test multiple groups (e.g. w or w/o batch_norm, different num_layer, different num_post_layer, etc.)